src/templates/armis_import.html

```diff
@@ -10,14 +10,26 @@ <h1>Armis Import</h1>
   <div id="formContainer" style="display: {% if display %}block{% else %}none{% endif %}">
     <form id="siteSubmitForm" method="POST" action="{% url 'armis_import' %}">
         {% csrf_token %}
-        <div class="checkbox-container">
-            {% for site_id, site_info in armis_sites.items %}
-                <div class="checkbox-item">
-                    <input type="checkbox" id="site-{{ site_id }}" name="site-ids[]" value="{{ site_id }}" class="custom-checkbox">
-                    <label for="site-{{ site_id }}" class="checkbox-label">{{ site_info.name }}</label>
-                </div>
+        <div class="split-content">
+          <div class="checkbox-container">
+              {% for site_id, site_info in armis_sites.items %}
+                  <div class="checkbox-item">
+                      <input type="checkbox" id="site-{{ site_id }}" name="site-ids[]" value="{{ site_id }}" class="custom-checkbox">
+                      <label for="site-{{ site_id }}" class="checkbox-label">{{ site_info.name }}</label>
+                  </div>
+              {% endfor %}
+          </div>
+          {% if vlan_blacklist %}
+          <div class="vlan-blacklist-box">
+            <strong>VLAN Blacklist:</strong>
+            <ul>
```

Comment:
A list object seems to be a bad idea. In productive use the blacklist will contain 20+ items causing a lot of scrolling.
You could also think about [collapse Object](https://getbootstrap.com/docs/5.3/components/collapse/) so that the vlans can be displayed on demand.